test/box-luatest/extra_assertions.lua

```diff
@@ -0,0 +1,11 @@
+local t = require('luatest')
+
+-- XXX: This is temporary. The assertion is present in latest Luatest and
+-- is making its way into Tarantool currently.
+t.assert_error_covers = function(expected, fn, ...)
```

Comment:
This helps looks trivial. Let's merge it into luatest now, before committing this patch.